README.md

```diff
@@ -103,18 +100,36 @@ require additional configurations applied to
 ### Prerequisites
 
 The following prerequisites are required to use the helm chart:
+- [Helm 3](https://helm.sh/docs/intro/install/) (Helm 2 is not supported) 
+- See https://github.com/kubernetes/helm for more information.
```

Comment:
This was a list of prerequisites: installed Helm 3 and Kubernetes cluster. It doesn't seem to be a list of prerequisites anymore.

Can we move the following line from the bottom here
- Administrator access to your [Kubernetes cluster](https://kubernetes.io/) and familiarity with your Kubernetes configuration. You must know where your log information is being collected in your Kubernetes deployment.

instead of this extra helm link?